$NetBSD: patch-server_CMakeLists.txt,v 1.2 2025/09/30 06:49:37 adam Exp $

Use correct variable: Boost_INCLUDE_DIRS.
Install config files to examples, per pkgsrc conventions.

--- server/CMakeLists.txt.orig	2025-09-23 19:02:05.000000000 +0000
+++ server/CMakeLists.txt
@@ -35,7 +35,7 @@ set(SERVER_LIBRARIES ${CMAKE_THREAD_LIBS
 
 include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_SOURCE_DIR}/server)
 
-include_directories(SYSTEM ${Boost_INCLUDE_DIR})
+include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 
 if(ANDROID)
   find_package(vorbis REQUIRED CONFIG)
@@ -155,8 +155,8 @@ else()
   install(
     FILES etc/snapserver.conf
     COMPONENT server
-    DESTINATION ${CMAKE_INSTALL_SYSCONFDIR})
-  install(DIRECTORY DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/snapserver/certs)
+    DESTINATION ${CMAKE_INSTALL_DATADIR}/examples/snapcast)
+  install(DIRECTORY DESTINATION ${CMAKE_INSTALL_DATADIR}/examples/snapcast/snapserver/certs)
   install(
     FILES etc/index.html
     COMPONENT server
